packages/vite/src/node/plugins/index.ts

```diff
@@ -61,12 +61,13 @@ export async function resolvePlugins(
     ),
     wasmPlugin(config),
     webWorkerPlugin(config),
-    workerImportMetaUrlPlugin(config),
     assetPlugin(config),
     ...normalPlugins,
     definePlugin(config),
     cssPostPlugin(config),
     config.build.ssr ? ssrRequireHookPlugin(config) : null,
+    isBuild && buildHtmlPlugin(config),
```

Comment:
Why do we also need to move this plugin?

Reply:
`assetImportMetaUrlPlugin` in build.pre, we need to make `workerImportMetaUrlPlugin` before `assetImportMetaUrlPlugin`

Reply:
Ok, lets check with vite-ecosystem-ci after merging 👍